enterprise/server/cmd/server/main.go

```diff
@@ -262,7 +267,7 @@ func main() {
 		log.Infof("Enabling memcache layer with targets: %s", mcTargets)
 		mc := memcache.NewCache(mcTargets...)
 		realEnv.SetCache(composable_cache.NewComposableCache(mc, realEnv.GetCache(), composable_cache.ModeReadThrough|composable_cache.ModeWriteThrough))
-	} else if redisTarget := configurator.GetCacheRedisTarget(); redisTarget != "" {
+	} else if redisTarget := configurator.GetDefaultRedisTarget(); redisTarget != "" {
```

Comment:
I think we want to keep the dedicated cache target here, but pull out the `SetMetricsCollector` and `SetKeyValStore` from this block and use the default target for those instead. That way, adding `app.default_redis_target` will not cause the Redis cache to be enabled (today, we intentionally don't specify `redis.cache_target` because we don't want to use the Redis cache).